mne/forward/forward.py

```diff
@@ -1230,28 +1228,31 @@ def apply_forward_raw(fwd, stc, raw_template, start=None, stop=None,
     --------
     apply_forward: Compute sensor space data and return an Evoked object.
     """
+    if isinstance(info, _BaseRaw):
+        warnings.warn('The "Raw_template" parameter is being deprecated '
+                      'and will be removed in MNE-0.11. '
+                      'Please provide info parameter instead',
+                      DeprecationWarning)
```

Comment:
Yeah, but since ``info`` is a required parameter, the call to this function would fail before capturing this case.